ios/RNLinksdk.m

```diff
@@ -190,11 +190,17 @@ - (void)stopObserving {
         }
     };
 
-
-    if ([publicTokenInput length] > 0) {
-        self.linkViewController = [[PLKPlaidLinkViewController alloc] initWithPublicToken:publicTokenInput
-                                                                            configuration:linkConfiguration
-                                                                                 delegate:self.linkViewDelegate];
+    if ([linkTokenInput length] > 0) {
+        if ([linkTokenInput hasPrefix:@"item-add-"]) {
```

Comment:
What's the difference between the `if` and `else` here?

Reply:
@zsweigart LinkKit has explicit and specific initializers, so `item-add-token`s must be passed to the <code>initWith**ItemAdd**Token</code> initializer whereas `public_token`s must use the <code>initWith**Public**Token</code> initializer.